src/dog.sol

```diff
@@ -59,9 +60,11 @@ contract Dog is LibNote {
     // --- Data ---
     struct Ilk {
         address clip;  // Liquidator
-        uint256 chop;  // Liquidation Penalty  [wad]
+        uint256 chop;  // Liquidation Penalty                                          [wad]
         uint256 hole;  // Max DAI needed to cover debt+fees of active auctions per ilk [rad]
         uint256 dirt;  // Amt DAI needed to cover debt+fees of active auctions per ilk [rad]
+        uint256 chip;  // Percentage of tab to suck from vat to incentivize keepers    [wad]
+        uint256 tip;   // Flat fee to suck from vat to incentivize keepers             [rad]
```

Comment:
I'd personally phrase the comments as "suck from vow" as that's where the `sin` is assigned.